templates/collectd.conf.j2

```diff
@@ -69,22 +68,22 @@ LoadPlugin syslog
 #	PrintSeverity false
 #</Plugin>
 
-#<Plugin log_logstash>
+{% if installed_version|version_compare("5.5", ">=") %}#<Plugin log_logstash>
 #	LogLevel info
 #	File "/var/log/collectd.json.log"
 #</Plugin>
-
-<Plugin syslog>
-	LogLevel info
-</Plugin>
+{% endif %}
+#<Plugin syslog>
+#	LogLevel info
+#</Plugin>
 
 ##############################################################################
```

Comment:
Remove everything from here to`Include "/etc/collectd.d"`.
It should be only comments.